include/compose.bash

```diff
@@ -216,6 +216,9 @@ cloudbreak:
         - CB_SMTP_SENDER_FROM=$CLOUDBREAK_SMTP_SENDER_FROM
         - CB_BAYWATCH_ENABLED=$CB_BAYWATCH_ENABLED
         - CB_BAYWATCH_EXTERN_LOCATION=$CB_BAYWATCH_EXTERN_LOCATION
+        - CB_CERT_DIR=/certs
```

Comment:
agree, it is usually bad idea to put anything under /tmp. maybe when we generate the cert on the fly from java code then this will be irrelevant when we start it from IDEA, because we can generate it under the 'IDEA project root'/cert